### PR TITLE
ps: added examples for current user processes

### LINKED ISSUE #1610: ps: add more CLI options
Need to add more command line options for ps command.

### PR DESCRIPTION
**Pull request by SubOptimal**
Fixes #1610

----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/ps.md [hunk 1]
```diff
@@ -13,3 +13,11 @@
 - Search for a process that matches a string:
 
 `ps aux | grep {{string}}`
+
+- List all processes of the current user in extra full format:
```
Comment (sbrl) [inline pages/common/ps.md]: What do you mean by "extra full format" here?

File: pages/common/ps.md [hunk 2]
```diff
@@ -13,3 +13,11 @@
...
+
+`ps --user $(id --user) -F`
+
+- List all processes of the current user as ASCII art process hierarchy:
```
Comment (sbrl) [inline pages/common/ps.md]: Perhaps `user as an ASCII art` would read better?

File: pages/common/ps.md (hunk 1 above)
Comment (agnivade) [inline pages/common/ps.md]: The extra full format is already covered by `auxww`. Let's just keep the next example.

File: pages/common/ps.md (hunk 2 above)
Comment (agnivade) [inline pages/common/ps.md]: How about changing it to - `List all processes of the current user as a tree:` ? And also, you can change `id --user` to `id -u` for brevity. As that is how its mentioned in the id.md page.

File: pages/common/ps.md (hunk 1 above)
Comment (SubOptimal) [reply pages/common/ps.md]: The explanation of `-F - extra full format` I took from the `man` page.

It's the same as `--format user,pid,ppid,c,sz,rss,psr,stime,tty,time,cmd`

| format specifier| description |
| --- | --- |
| user | effective user name |
| pid | process ID |
| ppid | parent process ID |
| c | processor utilization |
| sz | size in physical pages of the core image of the process |
| rss | size of the non-swapped physical memory |
| psr | processor that process is currently assigned to |
| stime | time the command started |
| tty | the terminal |
| time | cumulative CPU time |
| cmd | command with all its arguments |

Comment (SubOptimal) [reply pages/common/ps.md]: It's not really the same

using `auxww`

    USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
    suboptim    5503  0.0  0.1  22056  5888 pts/11   Ss   12:28   0:00 bash

using `-F`

    UID        PID  PPID  C    SZ   RSS PSR STIME TTY          TIME CMD
    suboptim  5503  3129  0  5514  5888   0 12:28 pts/11   00:00:00 bash

File: pages/common/ps.md (hunk 2 above)
Comment (SubOptimal) [reply pages/common/ps.md]: I will update it to `as a tree`.

But will keep `id --user` as this is suggested in https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#user-content-guidelines 
> Try to incorporate the **spelled-out version** of single-letter options in the example's description.

Comment (agnivade) [reply pages/common/ps.md]: No you did not misread. There are several implicit rules so its not practical to mention every one of them. The addendum to this rule is, if the single-letter version is easily understandable, then use that. In this case, `-u` can be easily remembered for `user`. But if it was something like `-m|--user`, then `--user` would be preferred. 

Hope that makes sense.

File: pages/common/ps.md (hunk 1 above)
Comment (agnivade) [reply pages/common/ps.md]: I see. It is indeed not a perfect match. What I meant was that it's more of an extended format like `auxww`. I wanted to have just one option with an extended format. But if you feel this is a helpful option from a beginner's perspective, we can keep it.

Comment (sbrl) [reply pages/common/ps.md]: Ah, ok. Sure, this works then 😺

File: pages/common/ps.md (hunk 2 above)
Comment (SubOptimal) [reply pages/common/ps.md]: Ok. Now it's more clear to me.

File: pages/common/ps.md (hunk 1 above)
Comment (SubOptimal) [reply pages/common/ps.md]: As the usage of `ps` is really divers I would suggest to add one more advanced example for beginner's, which shows how to find the parent process `PPID` easily.
I can't really judge if it is helpfull for most of the beginners or not. Why not leaving it in till someone propose an example maybe more useful for beginner's?

Comment (agnivade) [reply pages/common/ps.md]: Since we are now just at 5 examples, it should not be a big issue.

**Comment by sbrl** [comment]
Thanks for sticking with the review process, @SubOptimal :D

**Pull request merged**